web/src/test/java/nl/tehdreamteam/se42/web/token/generator/impl/SimpleTokenGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package nl.tehdreamteam.se42.web.token.generator.impl;
+
+import nl.tehdreamteam.se42.web.token.generator.AbstractTokenGeneratorTest;
+import nl.tehdreamteam.se42.web.token.generator.TokenGenerator;
+import nl.tehdreamteam.se42.web.token.generator.id.TokenIdGenerator;
+
+public class SimpleTokenGeneratorTest extends AbstractTokenGeneratorTest {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2187)